### PR TITLE
Add basic tests for dependencies

### PR DESCRIPTION
**Pull request by JacobCallahan**
These check some basic functionality for our top-level dependencies. They are not meant to be exhaustive, but rather to catch any obvious changes in behavior.
I also added a helper script for robottelo's config files. The initial helper allows a user to symlink their configs from another directory. We should expand on this script later for config migration etc.

### CODE REVIEW

**Comment by JacobCallahan** [comment]
Now would be a good time to add back in the robottelo tests to our GitHub checks. Or at the very least, the dependency tests.
Hold off on merge

**Comment by JacobCallahan** [comment]
@Gauravtalreja1 I commented out a test that is failing and may be related to #9481

**Review by mshriver — APPROVED** [review]

File: tests/robottelo/test_hammer.py [hunk 1]
```diff
@@ -109,189 +109,189 @@ def test_parsed_json_match_parsed_csv(self):
         assert hammer.parse_json(json_output) == hammer.parse_csv(csv_ouput_lines)[0]
 
 
-class TestParseHelp:
-    """Tests for parsing hammer help output"""
+# class TestParseHelp:
```
Comment (Gauravtalreja1) [inline tests/robottelo/test_hammer.py:112]: @JacobCallahan Fixing this test here in https://github.com/SatelliteQE/robottelo/pull/10375, so this failing test can be uncommented now

**Review by Gauravtalreja1 — CHANGES_REQUESTED** [review]

**Comment by jyejare** [comment]
@JacobCallahan This needs conflicts to resolve.

File: tests/robottelo/test_hammer.py (hunk 1 above)
Comment (JacobCallahan) [reply tests/robottelo/test_hammer.py]: @Gauravtalreja1 updated, thanks for addressing that.

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**